src/core/UI/base/typography.css

```diff
@@ -11,6 +11,16 @@
  * *****************************************************************************
  */
 
+
+
+[dir="ltr"] body, [dir="ltr"] div {
+    font-family: Roboto, sans-serif;
+}
+
+[dir="rtl"] body, [dir="rtl"] div {
+    font-family: "Noto Naskh Arabic", sans-serif;
```

Comment:
have you added a package for this to `package.json` as well?

Reply:
I wanted your perspective on this as we talked briefly about i18n last week.

Currently we use `import 'typeface-roboto'`, but there is no `typeface-noto-naskh-arabic` or similar available.

If we can control these fonts as small packages published on our _namespaced_ `@dhis2/typeface-${font-face}` repo we can add consistency at this layer.

Reply:
if we add a typeface package to the `@dhis2` scope we could move the `typeface-roboto` dep to that package as well and bundle all of our typefaces in a single package.

Reply:
sounds like a good idea